inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/recommender/NamedEntityLinkerClassificationToolFactory.java

```diff
@@ -38,15 +35,13 @@
 public class NamedEntityLinkerClassificationToolFactory
     implements ClassificationToolFactory<Object, Void>
 {
-    private final Logger log = LoggerFactory.getLogger(getClass());
+    public static final String PREFIX = "kb:";
```

Comment:
This should be consistent with the ConceptFeatureSupport. If we change it there, it must be modified here as well

Reply:
But referencing ConcepFeatureSupport introduces a cyclic dependency...

Reply:
Right ;) ConcepFeatureSupport probably needs to be moved from the kb-ui module to the kb-core module.